jaxrs/src/main/java/se/fortnox/reactivewizard/jaxrs/RequestLogger.java

```diff
@@ -69,6 +71,9 @@ public static String getHeaderValueOrRedact(Map.Entry<String, String> header) {
      * @return headers with sensitive information redacted
      */
     public static Set<Map.Entry<String, String>> getHeaderValuesOrRedact(Map<String, String> headers) {
+        if(headers == null) {
+            return null;
```

Comment:
Maybe we can return emptySet() here instead. Not sure the parameter is ever null or returning null will cause trouble but better safe than sorry